avm/res/cdn/profile/main.bicep

```diff
@@ -49,6 +49,9 @@ param ruleSets array = []
 @description('Optional. Array of AFD endpoint objects.')
 param afdEndpoints array = []
 
+@description('Optional. Array of Security Policy objects (see https://learn.microsoft.com/en-us/azure/templates/microsoft.cdn/profiles/securitypolicies for details).')
```

Comment:
@anderseide bare with me, another small one from my side 😉 
Since new functionality is added, could you please raise minor version in version.json? Thanks

Reply:
Done